nixos/modules/services/blockchain/ethereum/geth.nix

```diff
@@ -61,6 +61,35 @@ let
         };
       };
 
+      authrpc = {
+        enable = lib.mkEnableOption "Go Ethereum Auth RPC API";
+        address = mkOption {
+          type = types.str;
+          default = "127.0.0.1";
+          description = lib.mdDoc "Listen address of Go Ethereum Auth RPC API.";
+        };
+
+        port = mkOption {
+          type = types.port;
+          default = 8551;
+          description = lib.mdDoc "Port number of Go Ethereum Auth RPC API.";
+        };
+
+        vhosts = mkOption {
+          type = types.nullOr (types.listOf types.str);
+          default = ["localhost"];
+          description = lib.mdDoc "List of virtual hostnames from which to accept requests.";
+          example = ["localhost" "geth.example.org"];
+        };
+
+        jwtsecret = mkOption {
```

Comment:
`jwtSecret` would be a better name

Reply:
Sure.

Reply:
Wait, but other names don't use camel case, like `gcmode`, `syncmode`, or `maxpeers`...

Reply:
Hm, then, nevermind.

Reply:
All other arguments except for `extraArgs` don't use camel case, and this way they match how they are typed in the CLI flags themselves. Because of that I don't think it makes sense to use camel case just for that one.